Kitodo/src/main/java/de/sub/goobi/metadaten/Metadaten.java

```diff
@@ -81,27 +92,42 @@
 import org.kitodo.config.DefaultValues;
 import org.kitodo.config.Parameters;
 import org.kitodo.data.database.beans.Process;
+import org.kitodo.data.database.beans.Task;
+import org.kitodo.data.database.beans.User;
 import org.kitodo.data.database.exceptions.DAOException;
 import org.kitodo.data.exceptions.DataException;
 import org.kitodo.enums.PositionOfNewDocStrucElement;
 import org.kitodo.enums.SortType;
 import org.kitodo.legacy.UghImplementation;
 import org.kitodo.services.ServiceManager;
 import org.kitodo.services.file.FileService;
+import org.kitodo.workflow.Problem;
+import org.primefaces.PrimeFaces;
+import org.primefaces.event.DragDropEvent;
 import org.primefaces.event.NodeSelectEvent;
 import org.primefaces.event.TreeDragDropEvent;
 import org.primefaces.model.DefaultTreeNode;
 import org.primefaces.model.TreeNode;
 
 /**
  * Die Klasse Schritt ist ein Bean für einen einzelnen Schritt mit dessen
- * Eigenschaften und erlaubt die Bearbeitung der Schrittdetails
+ * Eigenschaften und erlaubt die Bearbeitung der Schrittdetails.
  *
  * @author Steffen Hankiewicz
  * @version 1.00 - 17.01.2005
  */
 public class Metadaten {
     private static final Logger logger = LogManager.getLogger(Metadaten.class);
+
+    /**
+     * Get imageHelper.
+     *
+     * @return value of imageHelper
+     */
+    public MetadatenImagesHelper getImageHelper() {
```

Comment:
Adding a method in list of method variable declaration should not go through any kind of review...